connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectService.scala

```diff
@@ -161,6 +168,54 @@ class SparkConnectService(debug: Boolean)
         userId = request.getUserContext.getUserId,
         sessionId = request.getSessionId)
   }
+
+  // customizedMethodDesc returns a customized MethodDescriptor
+  // with updated request and response marshallers.
+  private def customizedMethodDesc(
```

Comment:
nit: actually naming it methodWithCustomMarshallers will be self descriptive enough to likely not need the comment anymore
(which I know I asked for comments a moment ago, but now as I understand it, I see better how to make it self-descriptive without comments)